schema/defs.json

```diff
@@ -41,10 +41,10 @@
             "minimum": 0,
             "maximum": 18446744073709552000
         },
-        "percent": {
+        "maximum": {
             "type": "integer",
             "minimum": 0,
```

Comment:
You probably want to bump this from 0 to 1 to match [your Markdown docs][1].

[1]: https://github.com/opencontainers/runtime-spec/pull/777/files#diff-36e882271af2f3ad47923159cb746598R43